app/controllers/core_induction_programmes/core_induction_programmes_controller.rb

```diff
@@ -13,6 +13,8 @@ def index
   end
 
   def show
+    redirect_to guidance_question_path and return if current_user&.participant_profile&.show_guidance_speedbump?
+
```

Comment:
I would be tempted to keep this logic unchanged, and change where the link goes on the dashboard, depending on the flag in the db.

Being redirected immediately when you go to a page makes for slightly worse UX, I think.